### DISCUSSION TITLE
Fashion slider in Vue

### DISCUSSION THREAD

**Discussion by luethbas** [nolimits4web/swiper#5647] [Q&A]
Hi everyone,

is there any way to accomplish something like the following example with SwiperJS in Vue?
https://fashion-slider.uiinitiative.com/

Maybe someone else already tried that or can help me out with the transition used?

Edit: I cannot find any support or Patreon channel for UI iniative...

Thanks in advance!

**Comment by nolimits4web** [comment]
If you need support/help with UI Initiative items, use their support, or message them on Patreon
